server/src/test/java/org/spine3/server/entity/RecordBasedRepositoryShould.java

```diff
@@ -56,10 +59,10 @@
  */
 @SuppressWarnings("ConstantConditions")
 public abstract class RecordBasedRepositoryShould<E extends AbstractVersionableEntity<I, S>
-                                                          & TestEntityWithStringColumn,
-                                                  I,
-                                                  S extends Message>
-                                                  extends TenantAwareTest {
+        & TestEntityWithStringColumn,
```

Comment:
This is a compulsory measure in this case. The resulting type parameter must be:
 - of a type that `extends AbstractEntity`, e.g. `ProcessManager`;
 - of type  `TestEntityWithStringColumn `.

The only way to generify this was to use this weird syntax.

Can we leave this as is for now?